extensions/unicode-symbols/src/consants.ts

```diff
@@ -0,0 +1,5 @@
+import { raycastPreferences } from "./raycast-preferences";
+
+const defaultSearchResultLimit = 100;
+
+export const searchResultLimit = Number(raycastPreferences.searchResultLimitStr) || defaultSearchResultLimit;
```

Comment:
Since we don't support numeric preferences yet, I'm casting + defaulting them here.

Reply:
I'd suggest to cap this value to max value. The list view cannot handle unlimited entries – we recommend a limit of 1000 entries for lists (we do have extensions that use 2 to 3k entries and still work fine, though...)